code/processes/dqe.q

```diff
@@ -44,35 +44,35 @@ fillprocname:{[rs;h]
   if[count select from .dqe.results where id=idnum,procschk=proc,chkstatus=`started;
-    .dqe.failchk[idnum;"error:fail to complete before next run";proc]];
+    .dqe.failchk[man;idnum;"error:fail to complete before next run";proc]];
   }
 
-initstatusupd:{[idnum;funct;vars;rs]                                                                            /- set initial values in results table
+initstatusupd:{[man;idnum;funct;vars;rs]                                                                            /- set initial values in results table
   .lg.o[`initstatus;"setting up initial record(s) for id ",(string idnum)];
-  .dqe.dupchk[idnum]'[rs];                                                                                      /- calls dupchk function to check if last runs chkstatus is still started
-  `.dqe.results insert (idnum;funct;`$"," sv string raze (),vars;rs[0];rs[1];.z.p;0Np;0b;"";`started);
+  .dqe.dupchk[man;idnum]'[rs];                                                                                      /- calls dupchk function to check if last runs chkstatus is still started
+  `.dqe.results insert (idnum;funct;`$"," sv string raze (),vars;rs[0];rs[1];.z.p;0Np;0b;"";`started;man);
   }
 
-failchk:{[idnum;error;proc]                                                                                     /- general fail function, used to fail a check with inputted error message
+failchk:{[man;idnum;error;proc]                                                                                     /- general fail function, used to fail a check with inputted error message
   c:count select from .dqe.results where id=idnum, procschk=proc,chkstatus=`started;
   if[c;.lg.o[`failerr;raze "run check id ",(string idnum)," update in results table with a fail, with ",(string error)]];
-  `.dqe.results set update chkstatus:`failed,output:0b,descp:c#enlist error from .dqe.results where id=idnum, procschk=proc,chkstatus=`started;
+  `.dqe.results set update chkstatus:`failed,output:0b,descp:c#enlist error,manual:man from .dqe.results where id=idnum, procschk=proc,chkstatus=`started;
   }
 
-postback:{[idnum;proc;result]                                                                                   /- function that updates the results table with the check result
+postback:{[man;idnum;proc;result]                                                                                   /- function that updates the results table with the check result
   $["e"=first result;                                                                                           /- checks if error returned from server side
-  .dqe.failchk[idnum;result;proc];
-  `.dqe.results set update endtime:.z.p,output:first result,descp:enlist last result,chkstatus:`complete from .dqe.results where id=idnum,procschk=proc,chkstatus=`started];
+  .dqe.failchk[man;idnum;result;proc];
```

Comment:
Now that you're editing this line anyway can you fix the indent?

Reply:
Fixed it , thanks